crates/bevy-inspector-egui/examples/quick/world_inspector.rs

```diff
@@ -32,15 +32,15 @@ fn setup(
         ..default()
     });
     // light
-    commands.spawn(PointLightBundle {
+    /*commands.spawn(PointLightBundle {
```

Comment:
ditto

Reply:
`cargo test --all-features` and all `test result` came `ok`.